docs/en/integrations/deepnote.md

```diff
@@ -11,6 +11,11 @@ description: Efficiently query very large datasets, analyzing and modeling in th
 
 This guide assumes you already have a Deepnote account and that you have a running ClickHouse instance.
 
+## Interactive example
```

Comment:
@lkf24 this is great - thanks for adding this here.
@DanRoscigno , @rfraposa do we want to keep it here at the top or move it down?

Reply:
At first I thought it should be moved down, but then I tried it and saw how cool it was to have an interactive session in one click.  I think it is good up top.  People who already are familiar with Deepnote will skip down to the part that shows them how to integrate their existing Deepnote account with a ClickHouse DB, so I am good with it up top if you are @guykoh

Reply:
Fine by me. I approve this pr, but maybe we should wait to see if Rich prefer serving the image locally.

Reply:
I can update it later if needed.